src/SignIn.js

```diff
@@ -0,0 +1,248 @@
+  width: 70px;
+  border: none;
+  margin-top: 10px;
+  margin-left: 0;
+  padding-left: 0;
+  text-align: left;
+`;
+
+const ButtonSignUp = styled.button`
+  font-family: serif;
+  font-weight: bold;
+  color: white;
+  background: black;
+  border-radius: 15px;
+  text-align: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-top: 3px;
+  padding-bottom: 5px;
+  border: none;
+  margin-left: 10px;
+  margin-top: 15px;
+`;
+
+const RightBottomTag = () => {
+  return (
+    <Tag>
+      <TagImg src={process.env.PUBLIC_URL + './design/tag_2021.png'} />
+      <TagImg src={process.env.PUBLIC_URL + './design/tag_front.png'} />
+      <TagImg src={process.env.PUBLIC_URL + './design/tag_ceos.png'} />
+    </Tag>
+  );
+};
+
+export {
+  Top,
+  Container,
+  IndexImg,
+  RightBottomTag,
+  LabelBottom,
+  Input,
+  ButtonSignUp,
+};
+
+export default function SignIn() {
+  const [ID, setID] = useState();
+  const [PW, setPW] = useState();
+  const [loginData, setLoginData] = useState();
+  const [cookies, setCookie, removeCookie] = useCookies(['LoginData']);
+  let history = useHistory();
+
+  useEffect(() => {
+    setLoginData({
+      email: ID,
+      password: PW,
+    });
+  }, [ID, PW]); //setState 함수가 비동기로 걸리는데, 이가 왜 문제인지 알아보기
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    if (ID === null || PW === null) {
+      window.alert('값을 입력해주세요');
+      return false;
+    }
+
+    console.log(loginData);
+
+    setID('');
+    setPW('');
+
+    var config = {
+      method: 'post',
+      url: 'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/signin',
+      headers: {},
+      data: loginData,
+    };
+
+    axios(config)
+      .then(function (response) {
+        setCookie('LoginData', response.data);
+        console.log(response.data);
+        history.push('/vote');
+      })
+      .catch(function (error) {
+        console.log(error.response.data);
+      });
+  }
+
+  function handleChangeInput_ID(e) {
+    setID(e.target.value);
+  }
+
+  function handleChangeInput_PW(e) {
+    setPW(e.target.value);
+  }
+
+  const LoginForm = () => {
+    return (
+      <Form onSubmit={handleSubmit}>
+        <Label>ID</Label>
+        <Input type="email" value={ID} onChange={handleChangeInput_ID} />
+        <Label>Password</Label>
+        <Input type="password" value={PW} onChange={handleChangeInput_PW} />
+        <Button type="submit"> sign in ! </Button>
+      </Form>
+    );
+  };
+
+  const ClickToSignUp = () => {
+    return (
+      <Form>
+        <LabelBottom>* if you don't have an account</LabelBottom>
+        <LabelBottom>&nbsp;&nbsp;press the button below</LabelBottom>
+        <Link to={`/signup`}>
+          <ButtonSignUp> create account </ButtonSignUp>
+        </Link>
+      </Form>
+    );
+  };
+
+  return (
+    <Top>
+      <Container>
+        <IndexImg src={process.env.PUBLIC_URL + './design/index_mark.png'} />
+        <Box>
+          {LoginForm()}
```

Comment:
```suggestion
          <LoginForm />
```
함수형 컴포넌트를 만들었으니 이렇게 사용해도 되는 것으로 알고 있어요!